docs/data/material/customization/theming/theming.md

```diff
@@ -77,7 +77,18 @@ The community has built great tools to build a theme:
 
 ## Accessing the theme in a component
 
-You [can access](/system/styles/advanced/#accessing-the-theme-in-a-component) the theme variables inside your React components.
+You [can access](/system/styles/advanced/#accessing-the-theme-in-a-component) the theme variables inside your functional React components using the `useTheme` hook:
+
+```jsx
+import { useTheme } from '@mui/styles';
```

Comment:
```suggestion
import { useTheme } from '@mui/material/styles';
```

Reply:
Let's not reference the legacy styling solution. The util exported from `@mui/material/styles` is actually coming from [`@mui/system`](https://mui.com/system/basics/), which is a different one, based on emotion.